CryptoTechReminderSystem/Gateway/HarvestGateway.cs

```diff
@@ -83,5 +85,16 @@ public IList<TimeSheet> RetrieveTimeSheets(DateTimeOffset dateFrom, DateTimeOffs
                 }
             ).ToList(); 
         }
+        
+        private JObject RetrieveATimeSheet(DateTimeOffset dateFrom, DateTimeOffset dateTo, int page)
+        {
+            var address = $"/api/v2/time_entries?from={ToHarvestApiString(dateFrom)}&to={ToHarvestApiString(dateTo)}&page={page}";
```

Comment:
Could we extract `/api/v2/time_entries` and `/api/v2/users` into a constant?

Reply:
We could but not sure if it is necessary since we only use them once(?).

Reply:
Might make it easier to change if the api changes? 🤷‍♂️